packages/health-check/src/checks/critical.js

```diff
@@ -63,6 +76,25 @@ async function uploadCheck(done) {
   done({ name: "upload_file", time: calculateElapsedTime(time), ...data });
 }
 
+// skylinkHealthCheck checks if the skylink has reached full redundancy
+async function skylinkHealthCheck(skylink, numRetries, authCookie) {
+  try {
+    const response = await got(`https://${process.env.PORTAL_DOMAIN}/skynet/health/skylink/${skylink}`, {
+      headers: { cookie: authCookie },
```

Comment:
@kwypchlo how do I pass in a no cache option for this endpoint?

I tested that this code works with I add a manual sleep to get around the caching.

Reply:
I don't think there's a nocache option any more with skyd but cc @ChrisSchinnerl

Reply:
I'll just go with the adding the sleep back in for now.

Reply:
so I tested it and I can't target 10.10.10.10 directly. I get 

```js
  {
    name: 'upload_file',
    time: 2321,
    up: false,
    statusCode: 400,
    errorMessage: 'Response code 400 (Bad Request)',
    errorResponseContent: {
      message: 'Browser access disabled due to security vulnerability. Use Sia-UI or siac.'
    },
    ip: '10.10.10.10',
    errors: [
      {
        message: 'Response ip was different than current server ip - possibly there was an error with routing request',
        data: { response: '10.10.10.10', server: '66.29.147.77' }
      }
    ]
  },
```

I think if we can do your option of local traffic avoiding any default caching, that would be good

Reply:
I think we need to figure out the caching issue as it will lead to NDFs in the health checks.

I just tested these on my server and the large file upload passed but the small file upload failed. 

The small file upload got to 9x redundancy so was only sleeping for 1s in between requests so to me it seems like the cache wasn't updating and the test failed.

Reply:
> ```js
> Browser access disabled due to security vulnerability. Use Sia-UI or siac.
> ```

This error means that you need to pass "Sia-Agent" in "User-Agent" header. Not sure got allows that but that's what you should try.